server/src/types/TypesResolver.cpp

```diff
@@ -88,8 +89,10 @@ void TypesResolver::resolveStruct(const clang::RecordDecl *D, const std::string
     ss << "Struct: " << structInfo.name << "\n"
        << "\tFile path: " << structInfo.filePath.string() << "";
     std::vector<types::Field> fields;
-    fs::path sourceFilePath = sourceManager.getFileEntryForID(sourceManager.getMainFileID())->tryGetRealPathName().str();
-    for (const clang::FieldDecl *F : D->fields()) {
+    fs::path sourceFilePath = sourceManager.getFileEntryForID(
+            sourceManager.getMainFileID())->tryGetRealPathName().str();
```

Comment:
This logic is commonly used and can be moved to `ClangUtils`

Reply:
done